content/posts/fluidinfo-api-likes-and-dislikes.md

```diff
@@ -1,13 +1,12 @@
 ---
-layout: post
-title: "Fluidinfo API: Likes & Dislikes"
-tags:
- - fluidinfo
+date: "2011-04-24T00:00:00Z"
 published: true
-date: Apr 24th, 2011 9:01pm
+tags:
+- fluidinfo
+title: 'Fluidinfo API: Likes & Dislikes'
 ---
 
-I spent the last two weekends[1] working with the Fluidinfo API ([here is why]({{ site.url }}/2011/04/16/hello-fluidinfo-api-competition/)). Using Fluidinfo is pretty easy, I found two weekends enough to discover most of what is possible with Fluidinfo.[2] Here's my collection of likes and dislikes which accumulated while working with Fluidinfo. In the spirit of a true pessimist, I start with dislikes.
+I spent the last two weekends[1] working with the Fluidinfo API ([here is why](/2011/04/16/hello-fluidinfo-api-competition/)). Using Fluidinfo is pretty easy, I found two weekends enough to discover most of what is possible with Fluidinfo.[2] Here's my collection of likes and dislikes which accumulated while working with Fluidinfo. In the spirit of a true pessimist, I start with dislikes.
```

Comment:
Could you use [ref](https://gohugo.io/content-management/cross-references/#use-ref-and-relref) for the links to other pages?

Reply:
i think i removed these in a previous commit but then  reverted the commit during renaming files :(

Reply:
Sorry. But should be possible to do quickly with a regex replace. I am not sure about the `ref` syntax, but something like this:

```
search:   \[([^\]]*)\]\(/\d{4}/\d{2}/\d{2}/([0-9a-zA-Z-_/]*)\)
replace:  [$1]({{< ref "/post/$2" >}})
```

Reply:
i used a vim macro to do this ..  :)

Reply:
Vim macro, huh? 
https://giphy.com/embed/KpJ47gKe6b7v7xQyWj/video

Reply:
😆 .  Best thing with ref is  now you wont have broken internals urls . But there is a broken in whats-in-robotics-demo.md . which got caught .